code/modules/mob/living/simple_animal/hostile/giant_spider.dm

```diff
@@ -106,6 +108,16 @@
 	stop_automated_movement = 0
 	walk(src, 0)
 
+/mob/living/simple_animal/hostile/giant_spider/proc/validator_e_field(var/obj/effect/energy_field/E)
+	if(isliving(T)) // We prefer mobs over anything else
+		return FALSE
+	var/dist = get_dist(src, E)
+	if(dist < closest_distance)
+		closest_distance = dist
+		return TRUE
+	else

```

Comment:
Nitpick, you don't need this else statement at the end. Just return false if it fails. This is present throughout the PR.

Reply:
This false is required as it will be called several times on multiple shields, if we have 2 or more shields in our targets list, we want to target closest, and the way FindTarget() loops over this requires to return false, otherwise we have mobs targeting whichever shield was first in the list.

Reply:
That it’s not written in poor way, it is logically equivalent as there are only one costing that determines outcome.

Reply:
A stylistic choice. Also, arguably not an invalid one. A clear `if return A; else return B` indicates that both are of equal logical consequence. Where as a fall through would indicate that the `if return A;` is a special case.